[CI] ActiveDirectoryRunAsIT class failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #8904 / opensuse-leap-15_platform-support-unix](https://gradle-enterprise.elastic.co/s/rlgolyqsvffmi)
- [elasticsearch-pull-request #76432 / part-2](https://gradle-enterprise.elastic.co/s/ogbctxm3zn4c4)

**Reproduction Line:**
```
undefined
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.security.authc.ldap.ActiveDirectoryRunAsIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(org.elasticsearch.xpack.security.authc.ldap.ActiveDirectoryRunAsIT),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
undefined
```

**Issue Reasons:**
- [main] 2 failures in class org.elasticsearch.xpack.security.authc.ldap.ActiveDirectoryRunAsIT (0.4% fail rate in 506 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

```
java.lang.Exception: Suite timeout exceeded (>= 1200000 msec).
```

```
org.testcontainers.containers.ContainerLaunchException: Container startup failed for image es-smb-fixture:latest	
at __randomizedtesting.SeedInfo.seed([279354606BF92A34]:0)	
•••
Caused by: org.testcontainers.containers.ContainerFetchException: Can't get Docker image: RemoteDockerImage(imageName=<resolving>, imagePullPolicy=DefaultPullPolicy(), imageNameSubstitutor=org.testcontainers.utility.ImageNameSubstitutor$LogWrappedImageNameSubstitutor@105cd8f7)	
at org.testcontainers.containers.GenericContainer.getDockerImageName(GenericContainer.java:1367)	
•••
Caused by: com.github.dockerjava.api.exception.DockerClientException: Could not build image: failed to get digest sha256:c394f1624522be4d12bd7e689371c444ca73d4e0a7b7c4026298feed68e81b4c: open /var/lib/docker/image/overlay2/imagedb/content/sha256/c394f1624522be4d12bd7e689371c444ca73d4e0a7b7c4026298feed68e81b4c: no such file or directory	
at com.github.dockerjava.api.command.BuildImageResultCallback.getImageId(BuildImageResultCallback.java:78)	
•••
```

It failed with two different causes. Both of them seem to be test setup related, hence marking it as low risk.

--- a/x-pack/test/smb-fixture/src/main/java/org/elasticsearch/test/fixtures/smb/SmbTestContainer.java
+++ b/x-pack/test/smb-fixture/src/main/java/org/elasticsearch/test/fixtures/smb/SmbTestContainer.java
@@ -18,7 +18,7 @@ import java.time.Duration;
 
 public final class SmbTestContainer extends DockerEnvironmentAwareTestContainer {
 
-    private static final String DOCKER_BASE_IMAGE = "docker.elastic.co/elasticsearch-dev/es-smb-fixture:1.1";
+    private static final String DOCKER_BASE_IMAGE = "docker.elastic.co/elasticsearch-dev/es-smb-fixture:1.0";
     public static final int AD_LDAP_PORT = 636;
     public static final int AD_LDAP_GC_PORT = 3269;
 

--- a/x-pack/test/smb-fixture/src/main/java/org/elasticsearch/test/fixtures/smb/SmbTestContainer.java
+++ b/x-pack/test/smb-fixture/src/main/java/org/elasticsearch/test/fixtures/smb/SmbTestContainer.java
@@ -12,40 +12,18 @@ import com.github.dockerjava.api.model.Capability;
 import org.elasticsearch.test.fixtures.testcontainers.DockerEnvironmentAwareTestContainer;
 import org.testcontainers.containers.wait.strategy.Wait;
 import org.testcontainers.containers.wait.strategy.WaitAllStrategy;
-import org.testcontainers.images.builder.ImageFromDockerfile;
+import org.testcontainers.images.RemoteDockerImage;
 
 import java.time.Duration;
 
 public final class SmbTestContainer extends DockerEnvironmentAwareTestContainer {
 
-    private static final String DOCKER_BASE_IMAGE = "ubuntu:24.04";
+    private static final String DOCKER_BASE_IMAGE = "docker.elastic.co/elasticsearch-dev/es-smb-fixture:1.1";
     public static final int AD_LDAP_PORT = 636;
     public static final int AD_LDAP_GC_PORT = 3269;
 
     public SmbTestContainer() {
-        super(
-            new ImageFromDockerfile("es-smb-fixture").withDockerfileFromBuilder(
-                builder -> builder.from(DOCKER_BASE_IMAGE)
-                    .env("TZ", "Etc/UTC")
-                    .run(
-                        "DEBIAN_FRONTEND=noninteractive apt-get -o Acquire::Retries=10 update -qqy && apt-get install -qqy tzdata"
-                            + " winbind samba ldap-utils "
-                    )
-                    .copy("fixture/provision/installsmb.sh", "/fixture/provision/installsmb.sh")
-                    .copy("fixture/certs/ca.key", "/fixture/certs/ca.key")
-                    .copy("fixture/certs/ca.pem", "/fixture/certs/ca.pem")
-                    .copy("fixture/certs/cert.pem", "/fixture/certs/cert.pem")
-                    .copy("fixture/certs/key.pem", "/fixture/certs/key.pem")
-                    .run("chmod +x /fixture/provision/installsmb.sh")
-                    .cmd("/fixture/provision/installsmb.sh && service samba-ad-dc restart && echo Samba started && sleep infinity")
-                    .build()
-            )
-                .withFileFromClasspath("fixture/provision/installsmb.sh", "/smb/provision/installsmb.sh")
-                .withFileFromClasspath("fixture/certs/ca.key", "/smb/certs/ca.key")
-                .withFileFromClasspath("fixture/certs/ca.pem", "/smb/certs/ca.pem")
-                .withFileFromClasspath("fixture/certs/cert.pem", "/smb/certs/cert.pem")
-                .withFileFromClasspath("fixture/certs/key.pem", "/smb/certs/key.pem")
-        );
+        super(new RemoteDockerImage(DOCKER_BASE_IMAGE));
 
         addExposedPort(AD_LDAP_PORT);
         addExposedPort(AD_LDAP_GC_PORT);

--- a/x-pack/test/smb-fixture/src/main/resources/Dockerfile
+++ b/x-pack/test/smb-fixture/src/main/resources/Dockerfile
@@ -1,0 +1,16 @@
+FROM ubuntu:24.04
+
+ENV TZ="Etc/UTC"
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get update -qqy && \
+    apt-get install -qqy tzdata winbind samba ldap-utils
+
+COPY smb/provision/installsmb.sh /fixture/provision/installsmb.sh
+COPY smb/certs/ca.key /fixture/certs/ca.key
+COPY smb/certs/ca.pem /fixture/certs/ca.pem
+COPY smb/certs/cert.pem /fixture/certs/cert.pem
+COPY smb/certs/key.pem /fixture/certs/key.pem
+
+RUN chmod +x /fixture/provision/installsmb.sh
+
+CMD ["/bin/sh", "-c", "/fixture/provision/installsmb.sh && service samba-ad-dc restart && echo Samba started && sleep infinity"]

